packages/tools/devtools/devtools-core/src/index.ts

```diff
@@ -22,7 +22,15 @@
  */
 
 export { AudienceClientMetadata, MemberChangeKind } from "./AudienceMetadata";
-export { ContainerKey, FluidObjectId, HasContainerKey, HasFluidObjectId } from "./CommonInterfaces";
+export {
```

Comment:
Nit: would you mind ensuring that each of the edited export lists are alphabetized? I believe they were before.

Reply:
I *think* I have fixed them. I think I found some other places that don't have proper alphabetization, should I fix those too If I come across them?

Reply:
I would recommend just focusing on the spots that you changed. Keeping the PR diff minimal is also useful. If there are existing places that are unsorted, someone can always clean it up in a separate PR. And hopefully we'll get some linter rules in place for this eventually too.